components/Registration/index.tsx

```diff
@@ -148,6 +177,10 @@ const Registration: FunctionComponent<RegistrationProps> = ({
       },
     ]);
 
+    const _windowAction =
```

Comment:
don't start a variable with `_`

Reply:
Fixed in the latest commit as `airdropWindowAction`